lib/routes/account.js

```diff
@@ -229,34 +241,39 @@ module.exports = function (
           }
         }
 
+        function createKeyFetchToken () {
+          if (requestHelper.wantsKeys(request)) {
+            return password.unwrap(account.wrapWrapKb)
+              .then(
+                function (wrapKb) {
+                  return db.createKeyFetchToken({
+                    uid: account.uid,
+                    kA: account.kA,
+                    wrapKb: wrapKb,
+                    emailVerified: account.emailVerified
```

Comment:
There was discussion about making the keyFetchTokens also require verification here.  Just to be clear, we we deferring that to follow-up work because it's not needed for the desktop clients?  If so please file a follow-up issue so that we don't lost track of it.